frontend/mariner2/handle_rpm.go

```diff
@@ -48,6 +50,16 @@ func handleRPM(ctx context.Context, client gwclient.Client) (*gwclient.Result, e
 			return nil, nil, err
 		}
 
+		t := spec.Targets[targetKey]
+		if frontend.HasSigner(&t) {
+			signed, err := frontend.ForwardToSigner(ctx, client, platform, t.PackageConfig.Signer, st, "*.rpm")
```

Comment:
I don't think we can put "*.rpm" in here generically.
Also NOt sure this is explicitrly correct since the state should be a directory tree with `RPMS` and `SRPMS` in it.

Reply:
Instead this should probably be passed as part of the frontend config in the cmd line field.

Reply:
Do we not sign the SRPMs?

Reply:
We do need to sign SRPM's.
What I'm getting at with some of the suggestions is these should be passed as configuration (or by convention) to the signer.
What I mean by "by convention' here is the signer understands implicitly what its inputs are going to be and may not work with other inputs.

Reply:
Now irrelevant for this PR since this code has been moved elsewhere.

Reply:
I don't think this is irrelevant.
This is the argument we are passing into every signer, but the signer should already be aware of what is signing based on the target key.

Reply:
Mistook this for another issue. Definitely relevant. It's been addressed on the frontend side but forgot to make the corresponding change here. Thanks.

Reply:
Done.